src/trace/internal/utils.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TraceStateImpl } from './tracestate-impl';
+
+
+export function createTraceState(rawTraceState?: string): TraceStateImpl {
```

Comment:
Returning the interface only will prevent versioning issues if an internal/private property of the class is changed in the future. Users should only use the documented interface anyway.

```suggestion
export function createTraceState(rawTraceState?: string): TraceState {
```